slides/yocto-advanced/yocto-advanced.tex

```diff
@@ -308,8 +308,13 @@ \subsection{The power of BitBake}
       big. It is possible to clean old data with:
       \begin{block}{}
         \begin{minted}[fontsize=\footnotesize]{console}
-$ ./scripts/sstate-cache-management.sh --remove-duplicated -d \
-  --cache-dir=<SSTATE_DIR>
+$ ./scripts/sstate-cache-management.sh -y -d \
+  --cache-dir=$SSTATE_DIR
+        \end{minted}
+      \end{block}
+      \begin{block}{}
+        \begin{minted}[fontsize=\footnotesize]{console}
+$ find -atime +30 -delete $SSTATE_DIR
```

Comment:
This doesn't necessarily work on filesystem mounted with noatime which is very common on build machines so at least add a description of what this does